src/components/Popover/components/Trigger/Trigger.tsx

```diff
@@ -75,7 +81,9 @@ export const Trigger = ({
 
     const {onKeyDown} = useActionHandlers(handleClick);
 
-    return (
+    return typeof children === 'function' ? (
+        <React.Fragment>{children({onClick: handleClick, className, onKeyDown})}</React.Fragment>
```

Comment:
No need to wrap into fragment

Reply:
Sorry, there will be an error in Popover `undefined is not assignable to type Element | null`